website/docs/fun/counting_v2.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Counting System
+---
+
+This command controls the counting game and the statistics command.
+
+## Trigger
+
+**Type:** `Regex`<br />
+**Trigger:** `.+`
+
+:::note Restrictions
+
+Set this command to _only run_ in your counting channel in the channel restrictions.
+
+:::
+
+## Usage
+
+- Type a number, roman numeral, or a mix of both... with the option of math incrementing by 1, starting with 1, in the counting channel!
+- `-CountingStatistics` - Show server stats for counting
+- `-CountingStatistics <user>` - Show user stats for counting
+- `-CountingStatistics Leaderboard` - Show server leaderboard for number of times counted correct by each user
+
+:::tip Math Functionality
+
+To view all available math functions, see [Supported Functions, Operators, and Constants](https://github.com/ei14/calc#supported-functions-operators-and-constants)
+
+:::
+
+## Configuration
+
+- `$countTwice`<br />
+  Whether to allow users to count multiple times in a row.
+
+- `$correctRID`<br />
+  ID of role for most recent correct counting user.
+
+- `$incorrectRID`<br />
+  ID of role for incorrect counting users.
+
+- `$errorCID`<br />
+  ID of channel for any error output.
+
+- `$saves`<br />
+  Amount of saves from incorrect counts before the count resets.
```

Comment:
It's not clear what "saves from incorrect counts" means. Elaborate?

Reply:
The amount of times an incorrect number or expression can be made before the count resets.
If $saves is 1, the first incorrect number sent will get a pass, but the second will reset the count to 1.

Reply:
Thanks, but I meant that you should clarify the documentation.